### ISSUE TITLE
[Feature Request]: Rename 

### ISSUE THREAD

**Issue by jonaswre** [microsoft/AL-Go#586]

**Comment by freddydk** [comment]
@andersgMSFT - for comments

**Comment by andersgMSFT** [comment]
Hi Jonas 

Thanks for the feedback and sorry for the slow response!

I am not sure I understand the suggestion. So, I wanted to ask clarifying questions to make sure I have the right idea. 

We did look at the capability for our build tools to prefix the solutions with a tag (like debug/dev) etc. to allow multiple solutions to be deployed to the same environment, but we ran into some problems with the contend of the solution (id vs name), so we ended up abandoning that idea. 

Is the scenario you’re looking for something like that or more along the deployment tools supporting multiple solutions in one repository?

**Comment by jonaswre** [comment]
Hi, no worries. 

Yes I would like to prefix/postfix the solution name. But I wouldn't use dev or debug, I would use the company id. 

There is currently the issue - If you have 10 companies you need 10 Environments with each 1 solution. I would rather have 1 environment and 10 solutions. Thats more cost effective. 

I think this is a common pattern. If you want to use power apps to quickly deploy custom apps.

**Comment by jonaswre** [comment]
Yes making the BC Connector Mor flexible, would be the preferred solution. I don't really want the prefixing.
